lib/mediainfo-native/streams/general.rb

```diff
@@ -12,6 +12,7 @@ class GeneralStream < BaseStream
     mediainfo_attr_reader :overall_bit_rate, 'OverallBitRate'
     mediainfo_attr_reader :encoded_application_string, 'Encoded_Application/String'
     mediainfo_attr_reader :encoded_application, 'Encoded_Application'
+    mediainfo_attr_reader :is_streamable, 'IsStreamable'
```

Comment:
How about making it a `subject.general.streamable?` that returns a `boolean`?

e.g. 

```ruby
mediainfo_attr_reader :streamable?, "IsStreamable" do |value|
  value == "Yes"
end
```